js/request.js

```diff
@@ -0,0 +1,22 @@
+export function getCategories() {
+    const url = "http://localhost:8082/gift_certificates_system/tags?size=5";
```

Comment:
localhost should not be hardcoded. 
```suggestion
    const url = "/gift_certificates_system/tags?size=5";
```